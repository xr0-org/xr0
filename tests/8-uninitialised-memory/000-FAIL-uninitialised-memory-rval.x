void
main()
{
	int i;
	int j;

	i = j;
}
