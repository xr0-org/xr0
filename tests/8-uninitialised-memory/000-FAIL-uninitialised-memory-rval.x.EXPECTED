8-uninitialised-memory/000-FAIL-uninitialised-memory-rval.x:7:7: cannot exec statement: undefined memory access: `j' has no value
